packages/paste-core/components/form/stories/form.stories.tsx

```diff
@@ -4,29 +4,38 @@ import {Text} from '@twilio-paste/text';
 import {Box} from '@twilio-paste/box';
 import {Grid, Column} from '@twilio-paste/grid';
 import {Anchor} from '@twilio-paste/anchor';
-import {Checkbox, FormLabel, FormInput, Select, Option, Radio, FormTextArea, CheckboxGroup, RadioGroup} from '../src';
-import type {FormInputProps, SelectProps, FormTextAreaProps, CheckboxGroupProps, RadioGroupProps} from '../src';
+import {Checkbox, CheckboxGroup} from '@twilio-paste/checkbox';
```

Comment:
nitpick (non-blocking, out of scope): Keeping the `components/form` folder for a story file can be misleading to future developers of Paste. How would we feel about moving these to the respective package story files?

Reply:
I struggled with this. The reason we have this big story is more to capture the spacing between the different type of controls, vertically and horizontally. 

We had a collapsing margin bug once, so we need to ensure the vertical rythmn between all the form controls doesn't degrade. 

But that doesn't give it a good home as it uses all form components... 😬